docs/reference/customize_admin.rst

```diff
@@ -0,0 +1,65 @@
+Customize admin
+================
+
+There are few admins in sonata page bundle that can be customize using `Admin Extension`_
+As example if you want to add a new field in ``configureListFields``, You can do like this
+
+.. code-block:: php
+
+    namespace App\Admin\Extension;
+
+    use Sonata\AdminBundle\Admin\AbstractAdminExtension;
+    use Sonata\AdminBundle\Datagrid\ListMapper;
+    use Symfony\Component\DependencyInjection\Attribute\AutoconfigureTag;
+
+    #[AutoconfigureTag(name: 'sonata.admin.extension', attributes: ['target' => 'sonata.page.admin.page'])]
+    final class PageAdminExtension extends AbstractAdminExtension
```

Comment:
Isn't this doc enough https://docs.sonata-project.org/projects/SonataAdminBundle/en/4.x/reference/extensions/ ?